packages/terra-search-field/src/SearchField.jsx

```diff
@@ -34,12 +34,24 @@ const propTypes = {
    * A callback to indicate an invalid search. Sends parameter {String} searchText.
    */
   onInvalidSearch: PropTypes.func,
+
+  /**
+   * Indicates whether or not the search control should be disabled.
+   */
+  isDisabled: PropTypes.bool,
+
+  /**
+   * Indicates what sizing style should be used. Should be `fixed` or `stretch`.
+   */
+  sizing: PropTypes.string,
```

Comment:
could we set this to only allow for fixed or stretch like so?
sizing: PropTypes.oneOf(['fixed', 'stretch']),

Reply:
Definitely.... I should have thought of that.  Would you envision that the test for 'renders a search field with no stretch class specified if sizing invalid' is still needed with this specified?

Reply:
Actually, I went ahead and made the change and jest fails in situations where there's an invalid prop.  I'll remove the test as well.

Reply:
This has been addressed